src/Workspaces/CSharp/Portable/Formatting/Rules/SpacingFormattingRule.cs

```diff
@@ -377,7 +377,7 @@ public override AdjustSpacesOperation GetAdjustSpacesOperation(SyntaxToken previ
 
                 if (recursivePatternType != null)
```

Comment:
Also: it seems risky to assume that a DeconstructionPatternClause must be parented by a RecursivePatternSyntax.  That appears to be hte case now, but it's unclear why this would have to be the case in the future.

Reply:
This should be resolved now.